README.md

```diff
@@ -127,59 +127,26 @@ It's possible to package the repo's example application with `electron-builder`.
 
   The configured Linux packaging(s) will be generated in folder `examples/electron/dist`
 
-## Tests
-
-To run tests once, at the root of the project do:
-
-```shell
-yarn test --verbose
-```
-
-To keep tests running do:
-
-```shell
-yarn test --verbose --watch
-```
-
-### Test coverage
-
-The following command prints a coverage report to the terminal. As of now it covers all typescript files of the project, including those that are not supposed to have tests.
-
-```shell
-yarn test --coverage --collectCoverageFrom='src/**/*.ts'
-```
-
 ## Using the trace extension
-This section describes how to operate the Theia trace extension to view and analyze traces. The UI, view interactions, and UX are prelimiary and subject to revisions in the future.
+This section describes how to operate the Theia trace extension to view and analyze traces. The UI and view interactions are prelimiary and subject to revisions in the future.
 
 ![theia-trace-extension](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-0.0.2.png)
 
 ### Open a trace
-To open a trace in the Theia Trace Extension, use the **File Explorer** in Theia to navigate to the trace directory. Then right-mouse click on the trace and select menu **Open With->Open Trace**.
+When opening a trace, if the selection is a single file, then the tool will open the file directly as a trace.
```

Comment:
You removed the description, on how to open a trace. Please keep the description, but update the text `Open With-> Trace Viewer`. Also, the screen shots are not correct anymore and should reflect the latest.

Since there are multiple ways to open a trace, you could list all of them (with screenshots).

Reply:
I don't think it will harm the user experience to remove this content from the readme.

The new primary methods of opening a trace:
* Placeholder "Open Trace" button
* Trace widget "Open Trace" button
are discoverable ("self-documenting" UI) and do not need to be explained in the readme. Explaining these operations in the readme would be similar to writing comments to explain code that is already self-documenting (it adds unnecessary bulk).

My intuition is that the secondary "Open With --> Open Trace" method will almost never be used by users now that these new trace opening methods are available (apart from those who are already in the habit of opening traces this way). It is a corner case that I don't think we need to take up space explaining to users.

Reply:
@ssmagula Thoughts on whether trace opening needs to be explained in the readme?

Reply:
I argue to keep the option to open a trace from the file explorer, otherwise user won't know about that option. I think it's an important use case. Besides, this is the only way to open traces that are single files (non CTF), due to the directory chooser dialog.

Reply:
Ok good point! I'll add this back in and adjust the section. I didn't realize the dialog couldn't open single files.